app/src/main/java/com/example/myapplication/MatchData.java

```diff
@@ -59,28 +59,46 @@ public String returnAllData() {
     //Getters and Setters
 
     //Pre Match
-    public String getTeamNumber() {    return teamNumber;    }
-    public void setTeamNumber(String teamNumber) { this.teamNumber = teamNumber;    }
-    public String getMatchNumber() {    return matchNumber;    }
-    public void setMatchNumber(String matchNumber) { this.matchNumber = matchNumber;    }
+    public String getTeamNumber() {    
+        return teamNumber;    
+    
+    }
+    public void setTeamNumber(String teamNumber) { 
+        this.teamNumber = teamNumber;   
+    }
+    
+    public String getMatchNumber() {    
+        return matchNumber;    
+    }
+
+    public void setMatchNumber(String matchNumber) { 
+        this.matchNumber = matchNumber;    
+    }
 
     //Auto
-    public void setWorkingAuto(boolean workingAuto) {    this.workingAuto = workingAuto;    }
+    public void setWorkingAuto(boolean workingAuto) {    
+        this.workingAuto = workingAuto;    
+    }
     public void setMoveOutOfZone(boolean moveOutOfZone) {
         this.moveOutOfZone = moveOutOfZone;
     }
-    public boolean getMoveOutOfZone() {return moveOutOfZone;}
+
+    public boolean getMoveOutOfZone() {
+        return moveOutOfZone;
+    }
```

Comment:
Swap order for consistency.

Reply:
yep, that's one of the unnoticed issues, but something that I do want to you do is to clarify what should be swapped here since this doesn't explain what should be swapped and leaves a lot up to interpretation which you don't want when you're pointing out problems @MthreeJohnnyK

Reply:
Ok. Yeah, every time the order is get + set, but this one its set + get.